pyproject.toml

```diff
@@ -175,9 +175,9 @@ filterwarnings = [
     "ignore:\\s*Pyarrow will become a required dependency of pandas:DeprecationWarning", # Will go away by itself in Pandas 3.0
     "ignore:Passing a (SingleBlockManager|BlockManager) to (Series|GeoSeries|DataFrame|GeoDataFrame) is deprecated:DeprecationWarning", # https://github.com/holoviz/spatialpandas/issues/137
     "ignore:datetime.datetime.utcfromtimestamp():DeprecationWarning:dateutil.tz.tz", # https://github.com/dateutil/dateutil/pull/1285
-    # 2024-02
-    "ignore:The current Dask DataFrame implementation is deprecated:DeprecationWarning", # https://github.com/dask/dask/issues/10917
     # 2024-03
     "ignore:\\s*Dask dataframe query planning is disabled because dask-expr is not installed:FutureWarning",
     "ignore:numpy.ndarray size changed, may indicate binary incompatibility:RuntimeWarning", # https://github.com/pydata/xarray/issues/7259
+    # 2024-11
+    "ignore:The legacy Dask DataFrame implementation is deprecated and will be removed in a future version.:FutureWarning", # https://github.com/dask/dask/issues/10917
```

Comment:
Can you remove the dask stuff from the contest file and pixi.toml file? I think they were only there when Datashader did not directly support `dask-expr`.

https://github.com/holoviz/geoviews/blob/79cde9d60d15bafc1ad7621486caf4ee3450f7ab/geoviews/tests/conftest.py#L52-L57

https://github.com/holoviz/geoviews/blob/79cde9d60d15bafc1ad7621486caf4ee3450f7ab/pixi.toml#L13